src/Event/Http/FpmHandler.php

```diff
@@ -179,7 +179,8 @@ private function eventToFastCgiRequest(HttpRequestEvent $event): ProvidesRequest
         $request->setServerPort($event->getServerPort());
         $request->setCustomVar('PATH_INFO', $event->getPath());
         $request->setCustomVar('QUERY_STRING', $event->getQueryString());
-        $request->setCustomVar('LAMBDA_CONTEXT', json_encode($event->getRequestContext()));
+        $request->setCustomVar('REQUEST_CONTEXT', json_encode($event->getRequestContext()));
+        $request->setCustomVar('LAMBDA_CONTEXT', json_encode($context));
```

Comment:
Here is the BC break. Your suggested naming makes perfect sense, but we need to find an alternate naming.

Either we keep `LAMBDA_CONTEXT` and find another name. Or we deprecate `LAMBDA_CONTEXT` and move to a more consistent naming, for example:

- `AWS_LAMBDA_CONTEXT`
- `REQUEST_CONTEXT`

or another cheap trick like this?

Reply:
I would be happy with either approach, for ease deprecating `LAMBDA_CONTEXT` and replacing with `AWS_LAMBDA_CONTEXT` makes a lot of sense.